src/transport/Message.cpp

```diff
@@ -1,57 +1,47 @@
+#include <cstddef>
```

Comment:
I think it's because I removed an include from a header higher up the chain, and this is actually the lowest point that it's needed, but I'll double check.